enterprise/server/util/redisutil/redisutil.go

```diff
@@ -21,12 +24,67 @@ import (
 )
 
 var (
+	defaultRedisTarget          = flag.String("app.default_redis_target", "", "A Redis target for storing remote shared state. To ease migration, the redis target from the remote execution config will be used if this value is not specified.")
+	defaultRedisShards          = flagutil.StringSlice("app.default_sharded_redis.shards", []string{}, "Ordered list of Redis shard addresses.")
+	defaultShardedRedisUsername = flag.String("app.default_sharded_redis.username", "", "Redis username")
+	defaultShardedRedisPassword = flag.String("app.default_sharded_redis.password", "", "Redis password")
+
```

Comment:
I made this comment in another PR too, but wdyt about putting this in a separate package like `enterprise/server/util/redisconf` or similar? It would be nice to keep this package as a collection of generic redis utils, rather than having any BB-specific stuff

Reply:
I moved all this into `enterprise/server/backends/redis_client/redis_client.go`, per Tyler's suggestion. Same with the similar things in the cache aand remote_execution PRs ^-^

Thanks for letting me know about this, I didn't know about this goal of `util` before!